client/jetpack-onboarding/steps/contact-form.jsx

```diff
@@ -69,10 +69,12 @@ class JetpackOnboardingContactFormStep extends React.PureComponent {
 		const headerText = translate( "Let's grow your audience with Jetpack." );
 		const subHeaderText = (
 			<Fragment>
-				{ translate( "A great first step is adding Jetpack's contact form." ) }
+				{ translate(
+					'A great first step is adding a Contact Us page that includes Jetpack’s contact form.'
+				) }
 				<br />
 				{ translate(
-					'Create a Jetpack account to get started and unlock this and dozens of other features.'
+					'Create a Jetpack account to unlock this and dozens of other Jetpack features.'
```

Comment:
So we're saying 'Jetpack' three times now on two adjacent lines? 🤔 
('Create a Jetpack account to unlock this and dozens of other Jetpack features.' sounds particularly redundant, TBH.)

Reply:
Seems legit, @rickybanister @joanrho what do you think about this?

Reply:
I hear you—I had the same thoughts before. Some background: Rick and Growth emphasized that these specific features are exclusively brought to users by Jetpack. A user cannot get them without creating a Jetpack account.

I think the message could remain intact with a few "Jetpack" nixes like this:
**Let's grow your audience with Jetpack**
A great first step is adding a Contact Us page that includes a contact form.
Create a Jetpack account to unlock this and dozens of other features.

Reply:
That sounds great @joanrho

Reply:
Cool, update suggested in #22873 👍